README.md

```diff
@@ -50,9 +50,8 @@ rest = GerritRestAPI(url='http://review.example.net', auth=auth)
 changes = rest.get("/changes/?q=owner:self%20status:open")
 ```
 
-Note that is is not necessary to add the `/a/` prefix on the endpoint
-URLs. This is automatically added when the API is instantiated with an
-authentication object.
+Note that is is not necessary to add the `/a/` prefix; it is automatically
```

Comment:
```suggestion
Note that it is not necessary to add the `/a/` prefix; it is automatically
```

Reply:
Thanks.  Done in 8056e3bd6da9c8a67ef1311b59bee3c15ecf5df8